CHANGELOG.md

```diff
@@ -40,6 +40,8 @@ request adding CHANGELOG notes for breaking (!) changes and possibly other secti
 
 - Added a Management API endpoint to reset principal credentials, controlled by the `ENABLE_CREDENTIAL_RESET` (default: true) feature flag.
 
+- Added support for S3-compatible storage that does not have STS (use `stsUavailable: true` in catalog storage configuration)
```

Comment:
Could be worth expanding this sentence with a bit more of what's in the yaml for easy reference if people are reading the notes without wanting to read the full spec change -- specifically to clarify that it disables credential vending rather than vending some kind of root credentials after skipping STS subscoping.

How does the behavior compare to setting `SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION=true`?

Reply:
`SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION=true` disables all storage config in API responses. Disabling STS only removes credentials.

Reply:
Re: message changes: I'm not sure it's worth adding YAML to CHANGELOG... Would it not make it too verbose? In the end CHANGELOG has notes for many releases :thinking: 

How about I update it with a CLI example when I add CLI support for the new config entry?

Reply:
I agree that from an user standpoint, CLI (or `curl` :) ) and updating the configuration list on the documentation would be more than welcome.

Reply:
Yeah, to clarify, I didn't mean to add too much, just maybe adding a `; credential vending will then be disabled` in the parentheses so that the change note would say

    (use `stsUavailable: true` in catalog storage configuration; credential vending will then be disabled);

In any case, non-blocking.

Reply:
Incidentally, `SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION` really was only intended to pertain to the "credential" aspect, so it's unfortunate that it sort of fell into fully disabling StorageConfig mix-ins even for the non-credential-related config. But we can sort that out some other time.